src/client.ts

```diff
@@ -469,6 +491,22 @@ export class NuklaiVMClient {
     }
   }
 
+  public async getAssetBalance(address: string, assetAddress: string): Promise<string> {
```

Comment:
There's already a function `getBalance` that does the same thing as this. Do we wanna remove this and just the other one because I feel this is the same code.

Reply:
Do you mean adjust the `getBalance`, merging it into one yea?

Reply:
🫡